components/observations/ObservationsListView.tsx

```diff
@@ -160,10 +152,10 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
     [],
   );
 
-  if (incompleteQueryState(nacObservationsResult, nwacObservationsResult, mapResult) || !observations || !mapLayer) {
+  if (incompleteQueryState(observationsResult)) {
```

Comment:
We don't need to block on loading `mapResult` - we can make things load even if it fails (but it's always in cache so it doesn't).

Reply:
We will just pop in the zone names on the cards and into the filter dropdown? Can that induce jitter in the layout?

Reply:
never happens in practice, since this data has already fetched by the map on startup - but I can revert if it makes you happier.